source-academy/cadet-frontend
# File Explorer fix for #1787

### Description
Fix for #1787. File Explorer Dialog now closes when save is confirmed.

### Type of change

### How to test
- Open or save a new file. GitHub button text will change to reflect the file name.
- If changes are made in the editor after the latest open or save, the color of the file name text will change to orange. If there are no such unsaved changes, the color of the file name text will be blue.
- Additional dialog to confirm a save for a new file has been removed.
- Use the save-as option to create a new file. The file explorer should not linger.

### Checklist
